skills/vendure-cli/commands/migrate.md

```
@@ -0,0 +1,40 @@
+# vendure migrate
+
+Generates, runs or reverts TypeORM database migrations for a Vendure project.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)